meta/documents/user_guide/de/updating.md

```diff
@@ -0,0 +1,62 @@
+# Wayfair Plugin: Aktualisierung
+
+## Einführung
+Das Wayfair-Pluginwird  periodisch aktualisiert werden, um sicherzustellen, dass es den Lieferanten die bestmögliche Erfahrung bietet. **Plentymarkets aktualisiert das Wayfair-Plugin für Benutzer nicht automatisch**, daher müssen Lieferanten dies manuell aktualisieren.
+
+Lesen Sie vor dem Aktualisieren [die neuesten Informationen zu den Versionen des Wayfair Plugin](https://github.com/wayfair-contribs/plentymarkets-plugin/releases).
+
+## Aktualisierung auf die neueste Version des Wayfair Plugins
+1. Gehen Sie auf der Hauptseite von Plentymarkets zu `Plugins` >>` Plugin Set Übersicht`
+    * **Wählen Sie nicht die veraltete Option `Plugin-Übersicht.`**
+
+    * Auf dem Bildschirm sollte eine Liste der Plugin-Sets mit jeweils einer Reihe von Schaltflächen angezeigt werden.
+
+        ![plugins_menu_plugin_set_overview](../../../images/de/plugins_menu_plugin_set_overview.png)
+
+2. Suchen Sie das Plugin-Set, in dem das Wayfair-Plugin installiert ist.
+
+    * Die Liste der Plugins für das Plugin-Set ist hinter der Schaltfläche `Plugin-Set bearbeiten` versteckt.
+        ![linked clients](../../../images/de/plugin_sets_linked_clients.png)

```

Comment:
Compared to the English doc this section doesn't look correct and I'm not positive the correct way to fix it (although I'm sure the second screenshot is not aligned with the correct text). If this looks OK to you I can discuss on a call why I think it's fishy.